.coafile

```diff
@@ -22,3 +22,8 @@ bears = MarkdownBear, AlexBear
 
 [Git]
 bears = GitCommitBear
+
+[YML]
```

Comment:
Better if we use `[YAML ]`

Reply:
okay...
btw @aashutoshrathi what is the difference?

Reply:
YAML is the correct name.  YML is a DOS 3 letter extension some people use, but DOS is DEAD.